GVFS/GVFS.UnitTests/Mock/Common/MockPlatform.cs

```diff
@@ -14,7 +14,7 @@ namespace GVFS.UnitTests.Mock.Common
     public class MockPlatform : GVFSPlatform
     {
         public MockPlatform()
-            : base(executableExtension: ".mockexe")
+            : base(executableExtension: ".mockexe", installerExtension: ".exe")

```

Comment:
For consistency with `executableExtension` in the mock, update `installerExtension` to `".mockexe"` as well?

Reply:
Updated.